### ISSUE TITLE
mobile pre description is making display error

### ISSUE THREAD

**Issue by leccyril** [biati-digital/glightbox#430]
**Describe the bug**
on mobile mode the inner description is pasting title and text with no space making issue in the reading

**Are you able to reproduce the bug in the demo site**
Yes

**To Reproduce**
set inner description with <H4> tag and <p> tag

**Expected behavior**
Title should not taken in the resume of description

**Post the code you are using**
Help me solve issues faster, if I can copy paste your code I can try to reproduce and fix the bug faster.

**Screenshots**
here content of description :
![image](https://github.com/biati-digital/glightbox/assets/3234598/092d4e5b-806d-4701-96b4-8e15e504bad9)

and the first display we can see concatenated text

![image](https://github.com/biati-digital/glightbox/assets/3234598/47768f56-1290-4266-8833-0ae8aa6ee5ad)

**Smartphone:**
 any smartphone

**Comment by gingerchew** [comment]
@leccyril Can you add a codepen or similar? The description is stripped of HTML for compatibility and to prevent formatting errors.

**Comment by gingerchew** [comment]
Okay, let me rephrase. Can you give a codepen or something similar with a reduced test case?

**Comment by leccyril** [comment]
[[link here](https://codesandbox.io/p/sandbox/mutable-rgb-qcsjtj?file=%2Findex.html%3A18%2C1)
](https://codesandbox.io/p/sandbox/mutable-rgb-qcsjtj?file=%2Findex.html%3A1%2C1)

1. the mobile mode when we have to click "see more" to see the text is useless because we click on the picture then we only see title and 1 sentence..... we should be able to choose if we want this and on click directly display the content description
2. the style applied on the description will be also applied on the "see more" display so it is impossible to set style for the first display and then for the full display

we should be able to choose or not (just display full text directly)

**Comment by gingerchew** [comment]
Okay, I see now. What you are trying to do is not possible with the description property by design. You would want to use an inline slide to get the desired effect.
